frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -479,6 +482,31 @@ export function ClustersTable(props: {
 
   const rowActions = useMemo(() => [], [])
 
+  const advancedFilters = useMemo<ITableAdvancedFilter<Cluster>[]>(() => {
+    return [
+      {
+        id: 'name',
+        columnDisplayName: 'Name',
+        label: t('table.name'),
+        availableOperators: [SearchOperator.Equals],
+        tableFilterFn: (selectedValues, cluster) => {
+          const filtervalue = selectedValues[0]
```

Comment:
I don't love that we're assuming there's 1 entry in the array. I mentioned elsewhere this may be a consequence of trying to re-use the existing filter callback type, so I'm assuming you'll address this in iteration 2.

Reply:
That's correct actually, this filterFn callback changes in the subsequent iteration such that we are not assuming here...it was also bugging me haha.